tests/libs/__mocks__/prisma.ts

```diff
@@ -46,35 +45,6 @@ const handlePrismockBugs = () => {
     // @ts-ignore
     return __findManyWebhook(...rest);
   };
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  prismock.booking.findMany = (...rest: any[]) => {
-    // There is a bug in prismock where it considers `createMany` and `create` itself to have the data directly
-    // In booking flows, we encounter such scenario, so let's fix that here directly till it's fixed in prismock
-
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    const where = rest[0]?.where;
-    if (where?.OR) {
-      logger.silly("Fixed Prismock bug-3");
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      where.OR.forEach((or: any) => {
-        if (or.startTime?.gte) {
-          or.startTime.gte = or.startTime.gte.toISOString ? or.startTime.gte.toISOString() : or.startTime.gte;
-        }
-        if (or.startTime?.lte) {
-          or.startTime.lte = or.startTime.lte.toISOString ? or.startTime.lte.toISOString() : or.startTime.lte;
-        }
-        if (or.endTime?.gte) {
-          or.endTime.lte = or.endTime.gte.toISOString ? or.endTime.gte.toISOString() : or.endTime.gte;
-        }
-        if (or.endTime?.lte) {
-          or.endTime.lte = or.endTime.lte.toISOString ? or.endTime.lte.toISOString() : or.endTime.lte;
-        }
-      });
```

Comment:
Handled in a better way here https://github.com/calcom/cal.com/pull/11774/files#diff-8a93958cee6c90b2db64a192155f09e89cff776205469e1102a781b25c08ffecR272